sycl/CMakeLists.txt

```diff
@@ -139,6 +139,9 @@ install(DIRECTORY ${OPENCL_INCLUDE}/CL
 
 # Configure SYCL version macro
 set(sycl_inc_dir ${CMAKE_CURRENT_SOURCE_DIR}/include)
+set(sycl_src_dir ${CMAKE_CURRENT_SOURCE_DIR}/source)
+set(sycl_detail_inc_dir ${CMAKE_CURRENT_SOURCE_DIR}/include/CL/sycl/detail)
```

Comment:
Please, use paths relative to `sycl_inc_dir` to include headers from `include/CL/sycl/detail`.
```suggestion

```

Reply:
Sorry, I was thinking about changing C++ code from
```c++
#include <header.hpp>
```
to
```c++
#include <CL/sycl/detail/header.hpp>
```

Let's make this refactoring later, otherwise we will never commit this PR.

Reply:
Oh I see what you mean. That would be better. I'll do this in a followup PR.